site/guide/manage-users.qmd

```diff
@@ -63,17 +72,22 @@ To perform these next steps, you must be a group administrator or a superuser.
 
 4. Click the {{< fa xmark >}} in the pill next to the role for the user that you want to remove.
 
-## Viewing group members
+## User groups
+
+### View group members
 
 1. [Log in to the ValidMind UI]({{< var vm_ui >}}).
 
 2. Click **Settings**. 
 
 3. Under **User Permissions** in the sidebar, select **Groups**. 
 
-4. Click **See Details** to view the complete list of members in any group.
+4. Expand **See Details** to 
+   - view the complete list of members in any group
+   - scroll further down to view the list of models visible to this group
+
 
-## Adding or removing group members
+## Add or remove group members
```

Comment:
This is also missing a step between 2. and 3. You need to go to **Organization** first.

Reply:
Retested, your docs accurately reflect the UI.